### DISCUSSION TITLE
Getting Code Location Status From GraphQL

### DISCUSSION THREAD

**Discussion by jackwillisupside** [dagster-io/dagster#22913] [Q&A] [answered]
Hey! So I'm trying to create a op in our monitoring code location to identify if another code location goes down -- currently im hitting the Graph QL with this query below, but regardless of code location status in UI, (if its failed or not) I'm getting back that the code location is loaded. Am I just messing up this query and pulling the wrong identifier for the code location status?

```
            query CodeLocationQuery {
          locationStatusesOrError {
            ... on WorkspaceLocationStatusEntries {
              entries {
                name
                loadStatus
              }
            }
          }
        }
```

I've gotten status/an error back if I try to reload each code location, but this takes way longer and we'd rather just get that its down rather than retry to reload and get an error back.

**Comment by garethbrickman** [comment, marked as answer]
I believe it's this issue: https://github.com/dagster-io/dagster/issues/17571

There is a solution in this comment: https://github.com/dagster-io/dagster/issues/17571#issuecomment-2359051902
